src/components/auth/ForgotPassword.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
Please organize imports. External libraries first then internal utilities then internal components then assets and styles